roles/kubernetes/secrets/templates/openssl.conf.j2

```diff
@@ -26,3 +26,6 @@ IP.{{ 2 * loop.index }} = {{ hostvars[host]['ip'] | default(hostvars[host]['ansi
 {% set idx =  groups['kube-master'] | length | int * 2 + 1 %}
 IP.{{ idx }} = {{ kube_apiserver_ip }}
 IP.{{ idx + 1 }} = 127.0.0.1
+{% if kube_apiserver_ext_ip is defined %}
+IP.{{ idx + 1 | string }} = {{ kube_apiserver_ext_ip }}
```

Comment:
+1 for being `kube_apiserver_ext_ip` array as @mattymo suggested, plus here you need to use unique identifiers, e.g.:

```
IP.{{ idx + 1 }} = 127.0.0.1
{% if kube_apiserver_ext_ip is defined %}
IP.{{ idx + 2 | string }} = {{ kube_apiserver_ext_ip }} <-- notice 2 here
{% endif %}
```